README.md

```diff
@@ -109,6 +116,20 @@ done instead by editing the `config.yml` we used earlier.
 
 ## Step 4: Run the model
 
+## Bare metal
```

Comment:
We don't provide instructions of how to build bare metal above (e.g. `cd ./FV3; ./configure; ./compile`) so maybe the correct title for this section is `Manual` and not `Bare metal` since it feels more like instructions of how to manually run the model inside a docker container, as compared to actually running bare metal.

Reply:
I renamed this section  to  "Within container/manual" and the one below to "From host via docker".